src/api/custom-schema.ts

```diff
@@ -154,6 +155,10 @@ export const extendGraphqlSchema = (schema: GraphQLSchema) =>
         accountId: String
       }
 
+      type sendEmailInsuranceFeedbackResponse {
+        success: Boolean!
+      }
```

Comment:
Can we call this `SuccessResponse`? In a PR that may or may not be merged before this - cleaned up some of these duplicated sucess instances